variables.tf

```diff
@@ -74,24 +97,15 @@ variable "lacework_aws_account_id" {
   description = "The Lacework AWS account that the IAM role will grant access."
 }
 
-variable "global" {
-  type        = bool
-  default     = false
-  description = "Whether or not to create global resources. Defaults to `false`."
-}
-
-variable "regional" {
-  type        = bool
-  default     = false
-  description = "Whether or not to create regional resources. Defaults to `false`."
-}
-
 variable "iam_service_linked_role" {
   type        = bool
   default     = false
   description = "Whether or not to create aws_iam_service_linked_role. Defaults to `false`."
 }
 
+// The following inputs are considered deprecated.
+// Instead of providing these directly, the `global_module_reference` should supply them.
```

Comment:
nit: since terraform variables don't have a key to denote deprecation we could include it in the description. Perhaps with prefix description with `Deprecated`